### DISCUSSION TITLE
Every Glasgow School Mapped!

### DISCUSSION THREAD

**Discussion by ghost** [OpenOrienteering/mapper#1908] [Show and tell]
0. `16 Jun 2020` — https://twitter.com/BlairvadachOEC/status/1272897925107122176
  > Our orienteering maps are almost all complete for our hub schools.
  > We will host the completed images at https://www.blairvadach.org.uk/Pages/View/23
  > There will also be an `#OutdoorLearning` pack to support these maps very soon. It will contain ideas to enhance cross curricular use. `@EdISGlasgow`
  ![initial](https://pbs.twimg.com/media/Eao0FSYX0AEEeu0?format=jpg&name=medium)
1. **24 Feb 2021** — https://twitter.com/BlairvadachOEC/status/1364676083891724292
  > We've done it! Every school in Glasgow now has access to an orienteering style map of their school grounds.
  > Access your map here https://www.blairvadach.org.uk/Pages/View/30
  > Great for outdoor learning activities as well as games, sports and orienteering `#OpenOrienteering` `#Mapper` `#GCCRemoteLearning`
  > ![poster](https://pbs.twimg.com/media/EvBNqiPXAAggMqO?format=jpg&name=medium)
2. 25 Feb 2021 — https://twitter.com/Blairvadach_K/status/1364864146345451521
  > The team `@BlairvadachOEC` have worked hard during lockdown to create these, hopefully many pupils will get to use them over the coming years. `@IOLOutdoorProfs` `#outdoorlearning` `@scottish_o` `@GBRorienteering`
3. 12 Mar 2021 — https://twitter.com/GlasgowNPC/status/1370385779655114754
  > Orienteering maps for every school in Glasgow, paddling in Pinkerston - meet our fantastic new supporter `@BlairvadachOEC` `#NationalParkCity` https://www.glasgownationalparkcity.org/post/welcome-blairvadach-outdoor-centre
4. 12 Mar 2021 — https://twitter.com/MissProctor2/status/1370440837956521984
  > Also quick shout out to `@BlairvadachOEC` for their school map layouts. we had great fun using our compass knowledge to try some basic orientee and it was so much fun. Kids loved the maps and learned so much from them. Thank you 👏 👏
  > ![printed](https://pbs.twimg.com/media/EwTIpikWUAgbNnt?format=jpg&name=medium)
5. 16 Mar 2021 —https://twitter.com/StCharlesGla/status/1371897966479474691
  > P5 spent the afternoon in `@childrens_wood`. The children were learning how to orient a map, using features on the land to make sure their map was the right way round. They will develop these skills when they are introduced to `#Orienteering`. Thanks to `@BlairvadachOEC` for the map!

**Comment by tinnishill** [comment]
Just noticed this post. It raises three points.
1/ Most Glaswegian schools had already been mapped by STAG. Duplicating that work to what is apparently a lower standard is a bit odd.
2/ None of the maps I looked at (I haven't checked all of them) carry the OS copyright statement required by UK law and BOF regulation.
3/ The Orienteering community in West Central Scotland is quite small. No one from Blairvadoch has any presence in that community. Is what you are doing actually Orienteering ?

If you want to learn about Orienteering you will be made welcome at Orienteering events.

**Reply by dg0yt** [reply]
@tinnishill Probably you should contact Blairvadach Outdoor Education Centre directly. They may be unaware of this conversation.

**Reply by ghost** [reply]
I'm in no way related with *Blairvadach* or *STAG*, but I see some strange questions here:

> 1. Most Glaswegian schools had already been mapped by STAG. Duplicating that work to what is apparently a lower standard is a bit odd.

First of all, everyone is free to create own maps, free of some one else copyrights. I not see any "duplicate issues" if same areas would be mapped by some one else.

> 2. None of the maps I looked at (I haven't checked all of them) carry the OS copyright statement required by UK law and BOF regulation.

I just looked on *STAG Orienteering Club*maps and, as I see, their maps not include OS copyright marks too:

- https://www.stag-orienteering.co.uk/maps
  - https://www.stag-orienteering.co.uk/maps/linn-park

(FTR, I'm not an expert in how OS copyright statement requirements policy works in UK and Scotland)

> 3. The Orienteering community in West Central Scotland is quite small. No one from Blairvadoch has any presence in that community. Is what you are doing actually Orienteering?

As I see *Blairvadach Outdoor Education Centre* team organized many **un-official** (hobby-like) school orienteering events for scholar for years in style of outdoor training:

1. https://twitter.com/BlairvadachOEC/status/793858963519442944
2. https://twitter.com/BlairvadachOEC/status/819222341666357249
3. https://twitter.com/BlairvadachOEC/status/837266733891203073
4. https://twitter.com/BlairvadachOEC/status/968855137212039169
5. https://twitter.com/BlairvadachOEC/status/1309164847737851910
6. https://twitter.com/BlairvadachOEC/status/1438814807969382400
7. https://twitter.com/BlairvadachOEC/status/1565020079007436800

I suppose that it might be not a "real orienteering competition", but I not see any infringement that some one else doing some "orienteering-like" trainings & activity with self drawn orienteering maps.

Over all, *Blairvadach OEC* is NOT orienteering club & is not pretending to be members of BOF of IOF Sport.

BTW, I guess You may contact with *Blairvadach OEC* to ask any questions related to their training activity:

- https://www.blairvadach.org.uk/
